http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -298,6 +298,14 @@ object appQuery extends TableQuery(new AppTable(_)) {
   def markPendingDeletion(id: AppId): DBIO[Int] =
     updateStatus(id, AppStatus.Deleting)
 
+  def getAppsByNodepool(
+    nodepoolId: NodepoolLeoId
+  )(implicit ec: ExecutionContext): DBIO[Vector[GetAppsByNodepoolResult]] =
+    appQuery
+      .filter(_.nodepoolId === nodepoolId)
```

Comment:
Does this want to filter for non-deleted apps or something? Since presumably apps in `DELETED` status are already deleted in Sam.